dist/ng-token-auth.js

```diff
@@ -538,7 +538,7 @@ angular.module('ng-token-auth', ['ipCookie']).provider('$auth', function() {
                     _this.rejectDfd({
                       reason: 'unauthorized',
                       errors: resp.data != null ? resp.data.errors : ['Unspecified error']
-                    });
+                    }, resp.data != null);
```

Comment:
@dflourusso I dont think this is going to work how you think. I have not ran the code yet so i could be wrong, but during `validateToken` no matter what the response is you are going to pass a boolean value to `rejectDfd`. In `rejectDfd` you are checking if the input value for `invalidateTokens` is null and if so invalidate. Since you are passing a boolean type into `invalidateTokens`, true/false is never going to equal null. So the tokens will never be invalidated during `validateToken` now.